managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/OpAddEntry.java

```diff
@@ -201,7 +199,7 @@ public void closeComplete(int rc, LedgerHandle lh, Object ctx) {
     }
 
     private void updateLatency() {
-        ml.mbean.addAddEntryLatencySample(System.nanoTime() - startTime, TimeUnit.NANOSECONDS);
+        ml.mbean.addAddEntryLatencySample(System.nanoTime() - lastInitTime, TimeUnit.NANOSECONDS);
```

Comment:
I think both latencies could be reported. 
Right now, I think the addEntry is correct in that it's the time it takes to persist an entry, including the time of ledger append and the eventual queuing, if we don't have a ledger ready. 
It would be good to add a new metric to account just for the Ledger.addEntry() operation, though we shouldn't remove the current one since, in the end, it's the total time that matters to users.

Reply:
sure, added new metrics for ledger::addEntry